src/libraries/System.Diagnostics.EventLog/src/System/Diagnostics/Reader/EventBookmark.cs

```diff
@@ -15,15 +15,22 @@ namespace System.Diagnostics.Eventing.Reader
     /// well as marks the location in the result set of the EventReader
     /// that the event instance was obtained from.
     /// </summary>
-    public class EventBookmark
+    public sealed class EventBookmark
     {
-        internal EventBookmark(string bookmarkText)
+        /// <summary>
+        /// Creates a bookmark that identifies an event in a channel.
+        /// </summary>
+        /// <param name="bookmarkText">An XML string that represents the bookmark.</param>
+        public EventBookmark(string bookmarkText)
```

Comment:
The approved param name seems to have been "bookmarkXml" and the property was "BookmarkXml". Can you please match.